.github/workflows/ci.yml

```
@@ -13,6 +13,13 @@ concurrency:
 jobs:
   build-test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        profile: [ base, llama ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)